practicals/large_language_models.ipynb

```diff
@@ -16,7 +16,7 @@
         "\n",
```

Comment:
<p>Thanks for the changes in the text here! looks great!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>